bin/check-package-metadata.js

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env node
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback-next
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/**
+ * This is an internal script that will verify that all packages in our monorepo
+ * are registered in all required places and have the expected metadata in their
+ * package.json.
+ */
+'use strict';
+
+const path = require('path');
+const fs = require('fs-extra');
+
+const Project = require('@lerna/project');
+
+async function checkLicenseFiles(packages) {
+  for (const p of packages) {
+    if (!fs.existsSync(path.join(p.location, 'LICENSE'))) {
+      throw new Error(`${p.name} LICENSE file doesn't exist`);
+    }
+  }
+}
+
+async function checkMonorepoFile(packages, rootPath) {
+  const monorepoFile = await fs.readFile('docs/site/MONOREPO.md', 'utf8');
+
+  for (const p of packages) {
+    const packagePath = path.relative(rootPath, p.location);
+
+    const isPackageExist =
+      monorepoFile.includes(p.name) || monorepoFile.includes(packagePath);
+
+    if (!isPackageExist) {
+      throw new Error(
+        `MONOREPO.md doesn't contain mention about the ${p.name}`,
+      );
+    }
+  }
+}
```

Comment:
Can we also assert that the entires in MONOREPO.md are sorted lexicographically? I've encounter this problem recently when reviewing a pull request.

I am fine to leave such check out of scope of this pull request.

Reply:
@bajtos What do you think to generate `MONOREPO.md` dynamically before a new release? For example in the invocation of `npm run version` command

Reply:
I like the idea to generate `MONOREPO.md` automatically. I think we should update it as soon as a new package is added to the project and don't wait until a release happens. See `npm run update-ts-project-refs` for a similar automated step. Even with the automated generator in place, I think it's still a good idea to have a linter check run by CI, because sometimes people forget to commit the generated updates.

Reply:
@bajtos Ok, I'm ready to implement it after this PR